### ISSUE TITLE
Remove "my" and add "hand" in screen summary pieces (screen 2)

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#215]
Follow-up to #199
I think the "my" is awkward, and we should just call it a "challenge" like we do on screen 1. Adn I just noticed that "hands" is missing.

@zepumph, please adjust the dynamic screen description and hint phrases:
FROM
> Currently, **at the my challenge ratio** and somewhat close to each other:
> - Left hand near bottom of Play Area
> - Right hand just below middle of Play Area
> Explore **the current my challenge ratio** or create your own.

TO
> Currently, **hands at challenge ratio** and somewhat close to each other:
> - Left hand near bottom of Play Area
> - Right hand just below middle of Play Area
> Explore **current challenge ratio** or create your own.

**Comment by terracoda** [comment]
Looks good, thanks @zepumph.
